python/my_uploads.py

```diff
@@ -62,15 +62,15 @@ def list_my_uploaded_videos(uploads_playlist_id):
     maxResults=5
   )
 
-  print 'Videos in list %s' % uploads_playlist_id
+  print('Videos in list {uploads_playlist_id}')
```

Comment:
I believe there's a 'f' missing on the start of the print statement. Btw I have a branch that updated all the features to be compatible with python 3.8.x check it out at this link: https://github.com/EliasOlie/api-samples/tree/python3.8/python/python3.8.x, it may help you if you need some other functionality without the need to refactor the whole file to run ;)

Reply:
oh! I found the bug, thanks for the update